src/master.py

```diff
@@ -101,16 +102,28 @@ def download_if_changed(job_runner, local_path, ftp_host, ftp_path):
 job.add("wget -N https://raw.githubusercontent.com/ericminikel/minimal_representation/master/normalize.py")
 job.add("python -u normalize.py -R IN:%(reference_genome)s < IN:clinvar_table_raw.tsv > OUT:clinvar_table_normalized.tsv" % locals())
 
+#remove empty rows
+job.add("ex -s +'bufdo!v/\S/d' -cxa clinvar_table_normalized.tsv") #remove the empty lines
+#sort
+job.add("(cat IN:clinvar_table_normalized.tsv | head -1 > OUT:clinvar_record.tsv ) && " + # header row
+    "(cat IN:clinvar_table_normalized.tsv | tail -n +2 | egrep -v \"^[XYM]\" | sort -k1,1n -k2,2n -k3,3 -k4,4 >> OUT:clinvar_record.tsv ) && " + # numerically sort chroms 1-22
+    "(cat IN:clinvar_table_normalized.tsv | tail -n +2 | egrep \"^[XYM]\" | sort -k1,1 -k2,2n -k3,3 -k4,4 >> OUT:clinvar_record.tsv )")     # lexicogaraphically sort non-numerical chroms at end
+
+job.add("bgzip -c IN:clinvar_record.tsv > OUT:clinvar_record.tsv.gz")
+job.add("tabix -S 1 -s 1 -b 2 -e 2 IN:clinvar_record.tsv.gz", output_filenames=["clinvar_record.tsv.gz.tbi"])
+job.add("cp IN:clinvar_record.tsv.gz IN:clinvar_record.tsv.gz.tbi ../", output_filenames=["../clinvar_record.tsv", "../clinvar_record.tsv.gz", "../clinvar_record.tsv.gz.tbi"])
+
 # join information from the tab-delimited summary to the normalized genomic coordinates
-job.add("Rscript IN:join_data.R IN:%s" % variant_summary_table, input_filenames=['clinvar_table_normalized.tsv'], output_filenames=['clinvar_combined.tsv'])
+job.add("Rscript IN:join_data.R IN:%s" % variant_summary_table, input_filenames=['clinvar_record.tsv'], output_filenames=['clinvar_combined.tsv'])
```

Comment:
I think the variant_summary.txt file is still useful. Since it has the aggregated clinical_significance and review_status over all traits for the single allele, we could just make use of it by joining instead of computing from allele-trait table again.